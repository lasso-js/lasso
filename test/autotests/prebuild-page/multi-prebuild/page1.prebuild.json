[
  {
    "slots": {
      "body": "<script src=\"/static/page1/lasso$3.4.3/test/autotests/prebuild-page/multi-prebuild/b.js\"></script>"
    },
    "assets": [],
    "name": "page1"
  }
]