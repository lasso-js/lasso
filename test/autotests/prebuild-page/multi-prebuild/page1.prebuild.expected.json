[
  {
    "slots": {
      "body": "<script src=\"/static/page1/lasso$3.0.0-beta.10/test/autotests/prebuild-page/multi-prebuild/b.js\"></script>"
    },
    "assets": [],
    "name": "page1"
  }
]
